app/services/wishes/feedback_filterer.rb

```diff
@@ -32,20 +38,27 @@ def apply_stage_filter
     end
 
     def apply_wishlist_filter
-      return unless wishlist_id.present?
+      return unless board.present?
+
+      @scope = scope.search_by_board_name(board)
+    end
+
+    def apply_date_filter
+      return unless start_date.present? || end_date.present?
 
-      wishlist = Wishlist.find_by(id: wishlist_id)
-      @scope = scope.where(wishlist:) if wishlist
+      @scope = scope.where(created_at: (start_date.to_date)...(end_date.to_date + 1.day))
+    end
+
+    def apply_query_filter
+      return unless query.present?
+
```

Comment:
No need for space here

Reply:
rubocop ask for space after return keywords :')